.github/mergify.yml

```diff
@@ -0,0 +1,34 @@
+queue_rules:
+  - name: default
+    conditions:
+      - "check-success=test / test"
+      - "check-success=security/snyk (nationalarchives)"
+pull_request_rules:
+  - name: automatic merge for Scala Steward
+    conditions:
+      - author=scala-steward
+      - "check-success=test / test"
+      - "check-success=security/snyk (nationalarchives)"
+      - or:
+          - files=build.sbt
+          - files~=^(!?project/)
+    actions:
+      review:
+        type: APPROVE
+        message: Automatically approving Scala Steward
+      queue:
+        name: default
+  - name: automatic merge for dependabot
```

Comment:
Capitalise "Dependabot" for consistency?

Reply:
I've changed this one and the other name. The author field has to stay lower case as that's how it shows in GitHub.